### PR TITLE
Add births clients ratio plot type to time series

### PR DESCRIPTION
**Pull request by r-ash**

### CODE REVIEW

File: inst/metadata/time_series_plot_metadata.csv [hunk 1]
```diff
@@ -13,6 +13,7 @@ anc_clients,t_(ANC_CLIENTS),t_(ANC_CLIENTS_DESC),,"""0,0"""
 anc_tested,t_(ANC_TESTED),t_(ANC_TESTED_DESC),,"""0,0"""
 anc_tested_pos,t_(ANC_TESTED_POS),t_(ANC_TESTED_POS_DESC),,"""0,0"""
 anc_known_pos,t_(ANC_KNOWN_POS),t_(ANC_KNOWN_POS_DESC),,"""0,0"""
+births_clients_ratio,t_(BIRTHS_CLIENTS_RATIO),t_(BIRTHS_CLIENTS_RATIO_DESC"),,"""0%"""
```
Comment (rtesra) [inline inst/metadata/time_series_plot_metadata.csv:16]: This should be a count not percentage?

Comment (r-ash) [reply inst/metadata/time_series_plot_metadata.csv:16]: We've chosen to show other ratios as %, e.g. art_adult_sex_ratio and art_adult_child_ratio - I'm happy to change this to a count but just wanted to double check before doing so

**Comment by r-ash** [comment]
From ANC we return indicators `anc_clients`, `anc_known_pos`, `anc_already_art`, `anc_tested`, `anc_tested_pos`, `anc_known_neg`, `births_facility`. 

From ART we return indicators

But have metadata in `time_series_plot_metadata.csv` for id
art_total
art_adult_f
art_adult_m
art_adult
art_child
art_adult_sex_ratio
art_adult_child_ratio
art_prop_u15
anc_prevalence
anc_art_coverage
anc_clients
anc_tested
anc_tested_pos
anc_known_pos
births_clients_ratio
art_new_total
art_new_adult_f
art_new_adult_m
art_new_adult
art_new_child
vl_tested_12mos_total
vl_tested_12mos_adult_f
vl_tested_12mos_adult_m
vl_tested_12mos_adult
vl_tested_12mos_child
vl_suppressed_12mos_total
vl_suppressed_12mos_adult_f
vl_suppressed_12mos_adult_m
vl_suppressed_12mos_adult
vl_suppressed_12mos_child
vl_prop_suppressed_total
vl_prop_suppressed_adult_f
vl_prop_suppressed_adult_m
vl_prop_suppressed_adult
vl_prop_suppressed_child
vl_coverage_total
vl_coverage_adult_f
vl_coverage_adult_m
vl_coverage_adult
vl_coverage_child

**Comment by r-ash** [comment]
Have seen this working on dev
image

**Review by rtesra — APPROVED** [review]

**Pull request merged**